m02_setup/01_first_steps.rst

```diff
@@ -0,0 +1,190 @@
+.. The roles  should be created at the beginning of the document. Global roles can be created in the `global.rst` file
+
+.. include:: /global.rst
+
+.. Chapter content
+
+First steps
+===========
+
+.. styled-topic::
+
+  Main questions:
+      How to run the template course? and how to install the required software in your computer?
+
+  Topics:
+    In this section, we talk about:
+
+    * `Knowledge prerequisites`_
+    * `Software prerequisites`_
+    * `Cloning this codebase with git`_
+    * `Workflow`_
+
+  Material:
+    In this chapter, you won't need extra material.
+
+  Requirements:
+    You need basic computational skills to install some software in the Linux OS, and some knowledge on git.
+
+  Estimated working time:
+    From 30 min to 1 hour.
+    
+The `Aplus course Template <https://github.com/apluslms/aplus-course-template>`_ is a template that you will use to build your own course on the `A+ <https://github.com/apluslms/a-plus>`_ platform. You can also download the :abbr:`codebase (Codebase is a collection of source code used to build a particular software system, application, or software component.)` of this manual to understand better what is happening under the hood. Perhaps you could copy some of the directives we are using in your own course. The codebase of this manual is available in the GitHub `Aplus manual <https://github.com/apluslms/aplus-manual-dev>`_ repository.
```

Comment:
We prefer to avoid long lines even in RST text. Lines should be at most 80-100 characters long. At least you can divide long lines to multiple lines in the future. Of course, this does not affect the output of the RST compiler, but git tracks changes based on lines and it is easier to see what is changed when the lines are shorter. For example, if I add a comma here afterwards, the git diff would show that the whole paragraph has changed if it has been written on the same line.